etc/config/typescript.amazon.properties

```diff
@@ -2,18 +2,23 @@ compilers=&tsnative
 compilerType=typescript
 versionFlag=--version
 
-group.tsnative.compilers=tsc_0_0_26_gc:tsc_0_0_26_nogc
+group.tsnative.compilers=tsc_0_0_20_gc:tsc_0_0_20_nogc:tsc_0_0_26_gc:tsc_0_0_26_nogc
 group.tsnative.sharedlibs=
 group.tsnative.isSemVer=true
+compiler.tsc_0_0_20_gc.exe=/opt/compiler-explorer/tsc-0.0-pre-alpha20/bin/tsc
+compiler.tsc_0_0_20_gc.sharedlibs=/opt/compiler-explorer/tsc-0.0-pre-alpha20/lib/libTypeScriptRuntime.so
+compiler.tsc_0_0_20_gc.name=TypeScript Native Compiler 0.0.pre20 JIT (GC)
+compiler.tsc_0_0_20_gc.semver=0.0.20
+compiler.tsc_0_0_20_nogc.exe=/opt/compiler-explorer/tsc-0.0-pre-alpha20/bin/tsc
+compiler.tsc_0_0_20_nogc.name=TypeScript Native Compiler 0.0.pre20 JIT (No GC)
+compiler.tsc_0_0_20_nogc.semver=0.0.20
 compiler.tsc_0_0_26_gc.exe=/opt/compiler-explorer/tsc-0.0-pre-alpha26/bin/tsc
 compiler.tsc_0_0_26_gc.sharedlibs=/opt/compiler-explorer/tsc-0.0-pre-alpha26/lib/libTypeScriptRuntime.so
 compiler.tsc_0_0_26_gc.name=TypeScript Native Compiler 0.0.pre26 JIT (GC)
 compiler.tsc_0_0_26_gc.semver=0.0.26
-compiler.tsc_0_0_26_gc.alias=tsc_0_0_20_gc
```

Comment:
Presumably you want to be able to run the 0.0.20 and 0.0.26 side-by-side now?

Reply:
Yes, this is correct want to have both versions for now, as me last changes broke the setup somehow. I want to make sure that it is working with different versions

Reply:
Got it: thanks!